FreydCategoriesForCAP/gap/CategoryOfRows.gi

```diff
@@ -1251,54 +1254,10 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_CATEGORY_OF_ROWS,
     AddRandomMorphismWithFixedSourceAndRangeByInteger( category,
       function ( category, S, R, n )
         
-        if IsNegInt( n ) then
-          Error( "The input should be a non-negative integer\n" );
-        fi;
-        
```

Comment:
Please either keep this check or adjust the documentation.

Reply:
I have updated the documentation.